devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -29,8 +28,6 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0102", outClusters: "000A", manufacturer: "Feibit Co.Ltd", model: "FTB56-ZT218AK1.6", deviceJoinName: "Wistar Curtain Motor(CMJ)"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0102", outClusters: "000A", manufacturer: "Feibit Co.Ltd", model: "FTB56-ZT218AK1.8", deviceJoinName: "Wistar Curtain Motor(CMJ)"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0102", outClusters: "0003", manufacturer: "REXENSE", model: "DY0010", deviceJoinName: "Smart Curtain Motor(DT82TV)"
-		fingerprint manufacturer: "IKEA of Sweden", model: "KADRILJ roller blind", deviceJoinName: "IKEA KADRILJ Blinds", vid: "generic-shade-2" // raw description 01 0104 0202 00 09 0000 0001 0003 0004 0005 0020 0102 1000 FC7C 02 0019 1000
```

Comment:
I think for this PR we should just remove the fingerprints but leave everything else as it is. We'll need to migrate existing devices to the new DTH and then we can remove the unnecessary code in a follow up PR.

Reply:
That's right, i only removed ikea fingerprints and changed whitespaces (4 spaces -> tab) in zigbee-window-shade.groovy

Reply:
There are still a bunch of changes in this file that need to be left as is.

Reply:
> There are still a bunch of changes in this file that need to be left as is. Maybe you forgot to push your updates?

@tpmanley Anything specific ? I've just removed all the code related do ikea shades (including any supporting methods and other conditions).

Reply:
That's what I'm referring to. Ikea devices joined already will continue using this DTH so we can't remove the Ikea specific code until we run a script to migrate existing devices to the new DTH. The only changes to the existing DTH should be the removal of the fingerprints for now. In the future we can remove the Ikea specific stuff once the migration has been completed.

Reply:
ok, thanks for clarification.